ALVRTrackingInterface/ALVRTrackingInterface.cs

```diff
@@ -230,78 +246,86 @@ private void Update()
+            public static float LipPucker = 1.0f;
             public static float MouthX = 1.0f;
-            public static float MouthSmile = 1.22f;
-            public static float MouthFrown = 1.1f;
-            public static float LipFunnelTop = 1.13f;
-            public static float LipFunnelBottom = 8.0f;  //VERY NOT SENSITIVE
+            public static float MouthSmile = 1.0f;
+            public static float MouthFrown = 1.0f;
+            public static float LipFunnelTop = 1.0f;
+            public static float LipFunnelBottom = 1.0f;  //VERY NOT SENSITIVE
             public static float LipSuckTop = 1.0f;
             public static float LipSuckBottom = 1.0f;
-            public static float ChinRaiserTop = 0.75f;
-            public static float ChinRaiserBottom = 0.7f;
-            public static float MouthLowerDown = 2.87f;
-            public static float MouthUpperUp = 1.75f;
-            public static float MouthDimpler = 4.3f;
-            public static float MouthStretch = 3.0f;
-            public static float MouthPress = 10f;        //VERY NOT SENSITIVE
-            public static float MouthTightener = 2.13f;
-            public static float NoseSneer = 3.16f;
+            public static float ChinRaiserTop = 1.0f;
+            public static float ChinRaiserBottom = 1.0f;
+            public static float MouthLowerDown = 1.0f;
+            public static float MouthUpperUp = 1.0f;
+            public static float MouthDimpler = 1.0f;
+            public static float MouthStretch = 1.0f;
+            public static float MouthPress = 1.0f;        //VERY NOT SENSITIVE
+            public static float MouthTightener = 1.0f;
+            public static float NoseSneer = 1.0f;
         }
 
         // Preprocess our expressions per the Meta Documentation
-        private void UpdateEye()
+        private void UpdateEyeData(ref UnifiedEyeData eye, ref float[] expressions)
         {
+            #region Eye Data parsing
+
             // Recover true eye closed values; as you look down the eye closes.
             // from FaceTrackingSystem.CS from Movement Aura Scene in https://github.com/oculus-samples/Unity-Movement
             float eyeClosedL = Math.Min(1, expressions[(int)FBExpression.Eyes_Closed_L] + expressions[(int)FBExpression.Eyes_Look_Down_L] * 0.5f);
             float eyeClosedR = Math.Min(1, expressions[(int)FBExpression.Eyes_Closed_R] + expressions[(int)FBExpression.Eyes_Look_Down_R] * 0.5f);
 
             // Add Lid tightener to eye lid close to help get value closed
-            eyeClosedL = Math.Min(1, eyeClosedL + expressions[(int)FBExpression.Lid_Tightener_L] * 0.5f);
-            eyeClosedR = Math.Min(1, eyeClosedR + expressions[(int)FBExpression.Lid_Tightener_R] * 0.5f);
+            // eyeClosedL = Math.Min(1, eyeClosedL + expressions[(int)FBExpression.Lid_Tightener_L] * 0.5f);
+            // eyeClosedR = Math.Min(1, eyeClosedR + expressions[(int)FBExpression.Lid_Tightener_R] * 0.5f);
```

Comment:
Uncomment the EyeClosedL and EyeClosedR lines. EyeClosed for the Quest pro is more a confidence in that user has blinked so when closing the eyes slowly it is really hard to wink. The Lid Tightener, Squint, is required to help close the eyes. So this lines are important for proper eyelid tracking.

Reply:
Okay I'm looking through the code again and after implementing the reformulated v1 (SRanipal) EyeLidExpandedSqueeze combined parameter this lid tightener addition shouldn't be needed as adding lid tightener into eye openness will most likely cause issues with use in the new standard. 

I am using MeowFace and it works exactly as expected when using the new reformulated v1 parameter (I was having the same issue you were having with Quest Pro where the eye closes earlier than expected), the new formula will allow Squint to affect the eye blink and once the eyes are closed it will cause squint to be much more useable (which retains the behavior of both SRanipal eye squeeze and FACS/AR52 squint).

If you build an avatar to work with the new standard using the new parameters you also shouldn't have to worry about having the eyelid closing effectively as Squint will be completely separated from blinking and eye widen.

I'm just wondering if this addition of Lid Tightener is there because of the old parameters or if it's actually needed for the new standard. From my own testing it shouldn't be necessary.

Reply:
>// Lid Tightener is not tracked the same as SRanipal eye squeeze. This causes problems with combined parameters. The lid tightener has more controls the fine state of closing the eye while the eye lid is more of control blinking.
            // Eye close is non-linear and seems to be based on the confidence of that eye blink is detected. Lid tightener will be used to control the eye state thus squeeze will be disabled for now for the Quest Pro mapping.
            
These comments seem to indicate that it was for originally mapping into SRanipal base parameters. The combined parameters are the reason that SRanipal eye squeeze doesn't track as FACS/AR52 squint does. Mapping to the new standard should allow tightener to function as expected, especially with the reformulated EyeLidExpandedSqueeze.